style/ruby/.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+AllCops:
+  Exclude:
+    - 'Gemfile'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/spec_helper.rb'
```

Comment:
My reasoning at the time of doing this was that rails' and rspec's auto-generated code failed to pass the rubocop guidelines and it seemed strange to modify a bunch of auto-generated code knowing that it will fail out-of-the-box.

I could be convinced that the potential of somebody introducing problems in those files outweighs requiring them to modify auto-generated code in order to be in compliance.